pkg/daemon/ceph/util/util.go

```diff
@@ -53,5 +53,7 @@ func GetPortFromEndpoint(endpoint string) int32 {
 			logger.Errorf("failed to convert %q to integer. %v", portString, err)
 		}
 	}
-	return int32(port)
+	//#nosec G101 -- This is just a lower bit to bigger bit conversion
```

Comment:
The issue may just be that the rule id doesn't match
```suggestion
	//#nosec G109 -- This is just a lower bit to bigger bit conversion
```

Reply:
Thanks